packages/node_modules/@webex/webex-core/src/interceptors/auth.js

```diff
@@ -74,6 +74,10 @@ export default class AuthInterceptor extends Interceptor {
     const {resource, service, uri} = options;
     const {name} = details || {};
 
+    if ((service && service === 'web-authentication') || (name && name === 'web-authentication')) {
```

Comment:
also feels pretty hacky. let me know what you think, @InteractiveTimmy

Reply:
This works well, later, we will most likely make a change that pulls the auth validation interceptor directly into the service plugin to prevent this type of validation. For now, this works great. Be sure to add an additional test here: `packages/node_modules/@webex/webex-core/test/unit/spec/interceptors/auth.js L190`, you may need to modify the existing tests as well, if they are related to this change.